tests/integration_tests/build/test_binary_size.py

```diff
@@ -15,7 +15,7 @@
 SIZES_DICT = {
     "x86_64": {
         "FC_BINARY_SIZE_TARGET": 2520632,
-        "JAILER_BINARY_SIZE_TARGET": 902288,
+        "JAILER_BINARY_SIZE_TARGET": 965840,
```

Comment:
Do you have any ideas why the jailer binary size increased?

Reply:
I assume it is due to the updated dependencies.